src/js/_enqueues/wp/theme.js

```diff
@@ -1132,6 +1132,12 @@ themes.view.Themes = wp.Backbone.View.extend({
 		});
 	},
 
+	renderThemeCount: function( count ) {
+		this.liveThemeCount = count;
+		const currentLocale = document.documentElement.lang;
```

Comment:
For some reason this does not work when updating my profile language to Spanish:

English | Spanish
--- | ---
<img width="686" alt="Screenshot 2024-10-07 at 10 14 05" src="https://github.com/user-attachments/assets/dab96854-1a96-4282-a7c6-7ac0fe4a5858"> | <img width="688" alt="Screenshot 2024-10-07 at 10 14 43" src="https://github.com/user-attachments/assets/737c8b52-2092-455d-9b45-ff471fbfb499">

Reply:
According to the standards the correct way in Spanish is to only group thousands if the number is greater than or equal to 10000. See https://st.unicode.org/cldr-apps/v#/es/Symbols/70ef5e0c9d323e01

Checking, and this bears out, if I run:

```
const n = 8456;
console.log(
  n.toLocaleString('en'),
  n.toLocaleString('es'),
  n.toLocaleString('es-es'),
  n.toLocaleString('es-mx'),
);
```

I get: `"8,456", "8456", "8456", "8,456"` i.e. Spanish from Spain has no thousands separator

If I run:

```
const n = 98456;
console.log(
  n.toLocaleString('en'),
  n.toLocaleString('es'),
  n.toLocaleString('es-es'),
  n.toLocaleString('es-mx'),
);
```

I get: `"98,456", "98.456", "98.456", "98,456"` i.e. a thousands separator is present.

Reply:
> According to the standards the correct way in Spanish is to only group thousands if the number is greater than or equal to 10000

TIL!